src/styles/NavBar.css

```diff
@@ -61,8 +67,41 @@ span {
   color: var(--pp-selected-gray);
 }
 
+.nav-right-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.nav-button {
+  background-color: #ffffff;
+  color: var(--ks-font-medium);
+  border: 1px solid rgba(51, 51, 51, 0.25);
+  box-sizing: border-box;
+  border-radius: 30px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-top: 3%;
+  padding-bottom: 3%;
```

Comment:
I'd try to avoid exact percentages vs pixels as it scales poorly for smaller objects; what are you trying to achieve here?